packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/UI/UI.md

```diff
@@ -123,6 +126,14 @@ To provide users with a more specific direction, you can also include optional w
 
 ```
 
+### Welcome interaction
```

Comment:
Hmm I'm wondering if we could combine these 2 examples, since the demo also shows the prompts? 

Or, we could call this example "welcome message" and move it above the "welcome prompts" example? Then, remove the prompt cards from this demo. That way the focus is on the interaction of hiding the welcome message after a user message, and the welcome prompts" example makes more sense as a standalone.

Wdyt? Are these deviating too far from what you had in mind? Mainly, just wondering if we can streamline since the examples overlap a little 🤔

Reply:
+1 on having a welcome prompts as a standalone

Reply:
Pushing this now - let me know if you have feelings after it builds @edonehoo and @kaylachumley.